src/DevTools/DevTools.tsx

```diff
@@ -83,10 +58,49 @@ const DevToolsMain = ({
     setDevToolsOptions(options);
   }, [setDevToolsOptions, options]);
 
-  const theme_ = {
-    ...theme,
-    colorScheme,
-  };
+  const theme: MantineThemeOverride = React.useMemo(() => {
+    return {
+      primaryColor: 'dark',
+      activeStyles: { transform: 'scale(1)' },
+      fontFamily: 'Inter, -apple-system, BlinkMacSystemFont, Segoe, sans-serif',
+      fontFamilyMonospace:
+        'JetBrains Mono, ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, Liberation Mono, Courier New, monospace',
+      headings: {
+        fontFamily:
+          'Inter, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji',
+      },
+      defaultRadius: 'md',
+      globalStyles: (theme) => ({
+        '.jotai-devtools-shell': {
+          '*, *::before, *::after': {
+            boxSizing: 'border-box',
+          },
+          ...theme.fn.fontStyles(),
+          color: theme.colorScheme === 'dark' ? theme.white : theme.black,
+          lineHeight: theme.lineHeight,
+          WebkitFontSmoothing: 'antialiased',
+          MozOsxFontSmoothing: 'grayscale',
+          fontSize: theme.fontSizes.md,
+        },
+        [`.${shellTriggerButtonClassName}`]: {
+          position: 'fixed',
+          borderRadius: '50%',
+          borderWidth: 0,
+          width: '4rem',
+          height: '4rem',
+          zIndex: 99999,
+          img: {
+            height: '2rem',
+          },
+          left: position.includes('left') ? '0.2rem' : 'unset',
+          right: position.includes('right') ? '0.2rem' : 'unset',
+          top: position.includes('top') ? '0.2rem' : 'unset',
+          bottom: position.includes('bottom') ? '0.2rem' : 'unset',
```

Comment:
Could you add some tests for this? Could be as simple as checking for these positions based on the prop